src/main/kotlin/no/nav/syfo/senoppfolging/kafka/KSenOppfolgingSvarDTOV2.kt

```diff
@@ -0,0 +1,12 @@
+package no.nav.syfo.senoppfolging.kafka
+
+import no.nav.syfo.senoppfolging.v2.domain.SenOppfolgingQuestionV2
+import java.time.LocalDateTime
+import java.util.*
+
+data class KSenOppfolgingSvarDTOV2(
```

Comment:
Kan vurdere å ha alt eksternt som V1, da det blir rart å starte på v2 for det vi tilbyr.

På den andre siden er dette isyfo, så tror ikke verden går under om det starter på v2 heller :P

Reply:
Jeg fjerna heller "V2"